packages/actor-context-preprocess-statistic-link-discovery/lib/StatisticLinkDiscovery.ts

```diff
@@ -0,0 +1,61 @@
+import { StatisticBase } from '@comunica/bus-context-preprocess';
+import type { ILink } from '@comunica/types';
+
+export class StatisticLinkDiscovery extends StatisticBase<IDiscoverEventData> {
+  // Number of discover events tracked
+  public discoverEvents: number;
```

Comment:
```suggestion
  public events: number;
```
Would this still make sense?

Reply:
Yes, but it would lose some of its meaning. It annotates a discovery event with how many links were discovered before it. Would you think `numDiscoveredLinks` or something like this would be better? Or do you think `events` is clear enough given the name of the class?

Reply:
> Or do you think events is clear enough given the name of the class?

This was my reasoning indeed.

Reply:
Or just `count` could event work.